osc_ingest_trino/__init__.py

```diff
@@ -1,12 +1,3 @@
-"""
-osc-ingest-tools
-
-python tools to assist with standardized data ingestion workflows for the OS-Climate project
-"""
-
-# defines the release version for this python package
-__version__ = "0.3.1"
-
```

Comment:
@erikerlandson, I removed the docstring too since it was present in setup.py as well.